Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Dockerfile for Tilavarauspalvelu backend
+
+FROM python:3.8-buster as appbase
+
+RUN useradd -ms /bin/bash -d /tvp tvp
+
+WORKDIR /tvp
+
+# Can be used to inquire about running app
+# eg. by running `echo $APP_NAME`
+ENV APP_NAME tilavarauspalvelu
+
+# Served by whitenoise middleware
+ENV STATIC_ROOT /srv/static
+
+ENV PYTHONUNBUFFERED True
+
+RUN apt-get update && apt-get install -y postgresql-client gdal-bin
+
+RUN pip install --no-cache-dir uwsgi
+# Sentry CLI for sending events from non-Python processes to Sentry
```

Comment:
I am just curious: is the sentry CLI actually used somewhere?

Reply:
Not yet, but we'll use sentry in future so might as well.